### PR TITLE
Don't use scripts

### PR DESCRIPTION
**Pull request by SamYaple**
Also, add ref builds with build-args

### CODE REVIEW

File: debian/Dockerfile [hunk 1]
```diff
@@ -12,11 +16,29 @@ RUN set -x \
 		git \
 		libapache2-mod-wsgi \
 		virtualenv \
-	&& /downloader.sh \
-	&& /install.sh keystone \
+	&& DISTRO=debian \
```
Comment (intlabs) [inline debian/Dockerfile]: Move to ENV

File: debian/Dockerfile [hunk 2]
```diff
@@ -12,11 +16,29 @@ RUN set -x \
...
+	&& REPO=yaodu/requirements \
```
Comment (intlabs) [inline debian/Dockerfile]: Move to ARG

File: debian/Dockerfile [hunk 3]
```diff
@@ -12,11 +16,29 @@ RUN set -x \
...
+	&& TAG=latest-${DISTRO} \
```
Comment (intlabs) [inline debian/Dockerfile]: Move to ARG

Comment (SamYaple) [inline debian/Dockerfile]: DONE

**Comment by SamYaple** [comment]
Fixed issues, but moved to new PR https://github.com/yaodu/docker-keystone/pull/3

This should trigger dockerhub cloud build thing

**Pull request closed without merging**